### ISSUE TITLE
SCRC shortcut

### ISSUE THREAD

**Issue by claireh93** [ScottishCovidResponse/SCRCIssueTracking#214]
SCRCIssueTracking shortcut doesn't seem to link through to issues properly from PRs (i.e. for closing issues), or for commits.

**Comment by richardreeve** [comment]
Yes, this seems to be a bug in github. I think we may need to drop the shortcuts and just refer to ScottishCovidResponse/SCRCIssueTracking#214 instead, which is clunky, but I think / hope it works. Need to fix the documentation and send out an email though...
